firebase-dev.json

```diff
@@ -4,20 +4,25 @@
       "site": "yfthig-dev",
       "public": "dist",
       "ignore": ["**/devops/**", "**/.*", "**/node_modules/**"],
-      "headers" : [ {
-        "source": "**",
-        "headers" : [
-          { "key" : "Access-Control-Allow-Origin", "value" : "*" },
-          { "key" : "X-Frame-Options", "value" : "DENY" },
-          { "key" : "X-Content-Type-Options", "value" : "nosniff" },
-          { "key" : "X-XSS-Protection", "value" : "1; mode=block" },
-          {
-            "key": "Content-Security-Policy",
-            "value": "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.gov.bc.ca *.hotjar.com *.googleapis.com; style-src 'self' 'unsafe-inline' *.cloudflare.com *.googleapis.com *.jsdelivr.net; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com *.gstatic.com *.jsdelivr.net; img-src 'self' *.hotjar.com *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io *.apigee.net wss://*.hotjar.com *.hotjar.io; manifest-src 'self';"},
-          { "key": "Cache-Control", "value": "public, max-age=31536000"},
-          { "key": "Referrer-Policy", "value": "no-referrer" },
-          { "key": "Feature-Policy", "value": "microphone 'self'" }
-        ]
-      }]
+      "headers" : [
+        {
+          "source": "**",
+          "headers" : [
+            { "key" : "Access-Control-Allow-Origin", "value" : "*" },
+            { "key" : "X-Frame-Options", "value" : "DENY" },
+            { "key" : "X-Content-Type-Options", "value" : "nosniff" },
+            { "key" : "X-XSS-Protection", "value" : "1; mode=block" },
+            {
+              "key": "Content-Security-Policy",
+              "value": "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; style-src 'self' 'unsafe-inline' *.cloudflare.com *.googleapis.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com *.googleapis.com *.gstatic.com *.jsdelivr.net; img-src 'self' data: *.hotjar.com https://*.cac1.pure.cloud; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io wss://*.hotjar.com *.hotjar.io https://*.nr-data.net https://shyrka-prod-cac1.s3.ca-central-1.amazonaws.com https://*.newrelic.com https://*.cac1.pure.cloud wss://*.cac1.pure.cloud; manifest-src 'self'; media-src 'self' https://*.cac1.pure.cloud; object-src 'self' https://*.cac1.pure.cloud; child-src 'self' https://*.cac1.pure.cloud;"
```

Comment:
I see you've added the domains for HotJar and Maximus' webchat feature. Are those really required for this BC Registry web app?

Reply:
We may keep it consistency in all our apps.

Reply:
Hmm, ok, but stay tuned, because I've raised a yellow flag about all those Maximus resources and some people (Linda? Sarah? Rommel? Sinead? etc) are looking at it. It may be that we won't run remote scripts due to security/privacy concerns, but that's not decided yet.

cc: @JazzarKarim

Reply:
I will think about to put the "Content-Security-Policy" to 1password and generate the firebase json during deployment.